akka-stream-tests/src/test/scala/akka/stream/scaladsl/GraphMatValueSpec.scala

```diff
@@ -235,6 +235,25 @@ class GraphMatValueSpec extends StreamSpec {
       m4 should ===(NotUsed)
     }
 
+    "build more complicated graph with flows optimized for identity flows" in {
+      val flow1 = Flow.fromSinkAndSourceMat(Sink.ignore, Source.single(1).viaMat(Flow[Int])(Keep.both))(Keep.both)
+      val (mA, (m1, m2)) = Source.single(8).viaMat(flow1)(Keep.right).to(Sink.ignore).run()
+      Await.result(mA, 1.second) should === (Done) //from Sink.ignore
```

Comment:
`.futureValue` is a bit more concise than `Await.result` and picks up the default/right timeout from the test setup rather than using an arbitrary timeout everywhere